scriptmodules/emulators/dosbox.sh

```diff
@@ -54,6 +54,20 @@ function build_dosbox() {
     md_ret_require="$md_build/src/dosbox"
 }
 
+# helper method to ensure that 'freepats' soundfonts are configured for Timidity
+function _config_soundfonts_freepats_dosbox()
```

Comment:
Are you searching for the package name via dpkg? I don't think it's correct, as it's `fluid-soundfont-gm` and  `fluidr3_gm.cfg`.

Could you also use our helper functions for this as well as editing the file? i.e.:
`
# helper method to ensure that 'freepats' soundfonts are configured for Timidity
function _config_soundfonts_freepats_dosbox()
{
    hasPackage fluid-soundfont-gm && exit 0

    iniConfig " " "" "/etc/timidity/timidity.cfg"
    iniSet "source" "/etc/timidity/freepats.cfg"

    systemctl is-active --quiet timidity && systemctl restart timidity
}
`
The last line would also restart any running timidity daemon instance, just in case.

Reply:
Also forgot to mention: the `exit 0` will also prematurely end the configure_dosbox function if that package is installed, so it should return instead.

Reply:
I forgot about the `hasPackage` - thanks for the pointers.